### PR TITLE
january 20 discord gems

### PR DESCRIPTION
**Pull request by andronovhopf**

### CODE REVIEW

File: content/blog/2020-01-20-january-20-community-gems.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  - Discord
+  - DVC
+---
+
+## Discord gems
+
+There's a lot of action in our Discord channel these days. Ruslan, DVC's core
+maintainer, said it best with a gif.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">How it feels when <a href="https://twitter.com/DVCorg?ref_src=twsrc%5Etfw">@DVCorg</a> team is handling multiple conversations on Discord at the same time. <a href="https://t.co/QrLusdWYml">https://t.co/QrLusdWYml</a></p>&mdash; 🦉 Ruslan Kuprieiev (@rkuprieiev) <a href="https://twitter.com/rkuprieiev/status/1144008869414342658?ref_src=twsrc%5Etfw">June 26, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (shcheklein) [inline content/blog/2020-01-20-january-20-community-gems.md]: we merged @frontexpert 's solution, please merge master (if it's not yet) and then you should be able to just put a twitter link: 

eg:

```
tetx

https://twitter.com/rkuprieiev/status/1144008869414342658

text
```

File: content/blog/2020-01-20-january-20-community-gems.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  Some of those are related to our brainchild DVC and its journey. The others
+  are a collection of exciting stories and ideas centered around ML best
+  practices and workflow.
+picture: ../../static/uploads/images/2020-01-20/Community_Gems.png
+pictureComment:
+author: ../authors/elle_obrien.md
+commentsUrl: https://discuss.dvc.org/t/january-20-dvc-heartbeat/314
+tags:
+  - Discord
+  - DVC
```
Comment (shcheklein) [inline content/blog/2020-01-20-january-20-community-gems.md]: put Gems tag?

File: content/blog/2020-01-20-january-20-community-gems.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+descriptionLong: |
+  Every month we share news, findings, interesting reads,
+  community takeaways, and everything else along the way.
+  Some of those are related to our brainchild DVC and its journey. The others
+  are a collection of exciting stories and ideas centered around ML best
+  practices and workflow.
+picture: ../../static/uploads/images/2020-01-20/Community_Gems.png
+pictureComment:
+author: ../authors/elle_obrien.md
+commentsUrl: https://discuss.dvc.org/t/january-20-dvc-heartbeat/314
```
Comment (shcheklein) [inline content/blog/2020-01-20-january-20-community-gems.md]: fix the link?

File: content/blog/2020-01-20-january-20-community-gems.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+usefulness will depend on your particular needs.
+
+### Q: [How do I use DVC with SSH remote storage?](https://discordapp.com/channels/485586884165107732/563406153334128681/656016145119182849) I usually connect with a .pem key file. How do I do the same with DVC?
+
+DVC is built to work with the SSH protocol to access remote storage (we provide
+some
+[examples in our official documentation](https://dvc.org/doc/user-guide/external-dependencies#ssh)).
+When SSH requires a key file, try this:
+
+```
```
Comment (shcheklein) [inline content/blog/2020-01-20-january-20-community-gems.md]: use "```dvc .... ```" block to highlight the CLI syntax, "```python ... ```" for Python blocks

Comment (andronovhopf) [reply content/blog/2020-01-20-january-20-community-gems.md]: not sure i understand- if you have a minute to explain let me know

**Pull request merged**